src/barkhaPJun20/HighestTripletSum.java

```diff
@@ -0,0 +1,29 @@
+/*Find triplet having maximum sum.
+int[] num = {10,20,12,22,15,13,9,17,21}
+Max sum triplet is : 22,15,13 -> 50 */
+package barkhaPJun20;
+
+public class HighestTripletSum {
+	int highestSum = 0;
+	int sum;
+	String triplets;
+
+	public void SumOfTriplet(int[] num) {
+		for (int index = 0; index < num.length; index++) {
+			sum = num[index] + num[index + 1] + num[index + 2];
+
+			if (sum > highestSum) {
+				highestSum = sum;
+				triplets=num[index]+", " + num[index + 1]+", " + num[index + 2];
+			}
+			index += 2;
```

Comment:
Hi @BarkhaPJun20 
Excellent work,
Improvement part : Instead of increase value on line 19th you can add this in loop line 13th (instead of i++ you can write index+=3).